doc/design/volumes/nfsvolumestores.md

```diff
@@ -18,10 +21,36 @@ Allow the container user to
  2. Exposing shared storage configuration via VIC (e.g. IOPS per client, storage policy, etc.)
  3. Management of shared storage via VIC (e.g. container quiesce for storage maintenance, quota manipulation of the target, etc.)
 
+### Networking
+
+Containers will need to be on an appropriate network for the NFS volume store to be accessible.
+
+There are ways this could be done:
+ - allow association of volume-store with container-network to allow direct coupling
```

Comment:
Well, chatting with @hickeng on this, the reason we allow the user to pass volume information by name and not by _host:mountpoint_ is to hide the details of the underlying infrastructure.  The user shouldn't need to know what the VI admin configured the VIC deployment with and at what addresses/hosts the NFS targets are available.  Rather, the `docker` user need only specify the `VolumeStore` name.  And in that respect, the user _won't_ know the network topology and can't be expected to know what network the volume store is listening on.  

For the short term, we can do what you're suggesting.  But the longer term is to do this configuration automagically with microsegmentation so the container is configured to talk to the NFS target by adding it to the appropriate network and defining the IP rules insulating the container from the NFS network and vice versa.